view.php

```diff
@@ -144,12 +144,14 @@
 
 /// Print the main part of the page
 
-$summary = format_module_intro('etherpadite', $etherpadlite, $cm->id);
+echo $OUTPUT->heading($etherpadlite->name);
+
+$summary = format_module_intro('etherpadlite', $etherpadlite, $cm->id);
 if(isguestuser() && !etherpadlite_guestsallowed($etherpadlite)) {
 	$summary.= "<br/><br/>".get_string('summaryguest','etherpadlite');
 }
 if(!empty($summary)) {
-	echo $OUTPUT->box($summary, 'generalbox mod_introbox', 'etherpadliteintro');
+	echo $OUTPUT->box($summary, 'generalbox', 'intro');
```

Comment:
Hmm, when we use #intro, then a style from mod_wiki gets applied.
mod_introbox is used even in the base theme.
Why do you want to change it?